system/preside-objects/admin/security/security_user_two_factor_login_record.cfc

```diff
@@ -9,9 +9,7 @@
  * @feature   twoFactorAuthentication
  */
 component extends="preside.system.base.SystemPresideObject" displayName="User 2FA Login record" {
-	property name="security_user" relationship="many-to-one" relatedTo="security_user" required=true uniqueindexes="user_machine|1";
+	property name="security_user" relationship="many-to-one" relatedTo="security_user" required=true uniqueindexes="user_machine";
```

Comment:
The unique index has to go. A single user can be logged in to multiple devices, each with its own cookie so there could be multiple entries with the same user.